packages/neuron-ui/src/contexts/transfer.ts

```diff
@@ -0,0 +1,12 @@
+import { createContext } from 'react'
```

Comment:
If this really necessary to be a context?

Reply:
No. Context should be at minimum.

Reply:
If I want transfer data to be remembered, how to do ?

Reply:
The component should use local state/property to handle that. When any writings happen, they will pass through IPC channel to the electron app, and our global wallet gets updated automatically.

We'll implement viewing the wallet context as heart of the UI layer. At the moment keep in mind we just want to mock the UI flow.

Reply:
ok.I get it